### PR TITLE
task(ccls-1759): make pipeline helm-ready

### PR DESCRIPTION
**Pull request by keith-h**

### CODE REVIEW

File: .github/workflows/maven.yml [hunk 1]
```diff
@@ -53,39 +67,43 @@ jobs:
...
+    if: ${{ github.ref_name == 'development' }}
+    needs: [define-image-tag]
+    runs-on: ubuntu-latest
+    environment: development
+    steps:
+      - name: Checkout charts repo
+        uses: actions/checkout@v3
+        with:
+          repository: ministryofjustice/laa-ccms-caab-helm-charts
+          ref: task/create-helm-charts
```
Comment (PhilDigitalJustice) [inline .github/workflows/maven.yml]: Need to be 'development'

File: .github/workflows/maven.yml [hunk 2]
```diff
@@ -1,14 +1,28 @@
 # This workflow will build a Java project with Maven
 # For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
 
-name: CI Github
+name: CI caab-saml-mock
 
 on:
   push:
-    branches-ignore:
-      - master
```
Comment (PhilDigitalJustice) [inline .github/workflows/maven.yml:10]: Do we need to set this to run on main?

**Review by PhilDigitalJustice — CHANGES_REQUESTED** [review]
couple of comments

File: .github/workflows/maven.yml [hunk 3]
```diff
@@ -53,39 +67,43 @@ jobs:
...
-#        run: |
-#          echo "${{ secrets.KUBE_CERT_DEV }}" > ca.crt
-#          kubectl config set-cluster ${KUBE_CLUSTER} --certificate-authority=./ca.crt --server=https://${KUBE_CLUSTER}
-#          kubectl config set-credentials deploy-user --token=${{ secrets.KUBE_TOKEN_DEV }}
-#          kubectl config set-context ${KUBE_CLUSTER} --cluster=${KUBE_CLUSTER} --user=deploy-user --namespace=${{ secrets.KUBE_NAMESPACE_DEV}}
-#          kubectl config use-context ${KUBE_CLUSTER}
-#      - name: Apply the updated manifest
-#        run: kubectl -n ${KUBE_NAMESPACE} apply -f kubectl_deploy --namespace=laa-ccms-civil
+  update-helm-chart:
+    if: ${{ github.ref_name == 'development' }}
```
Comment (PhilDigitalJustice) [inline .github/workflows/maven.yml]: Are we not checking the git ref for the app repo here?

Shouldnt this be main? but if you change the actions 'on build ' step this isnt needed

File: .github/workflows/maven.yml (hunk 2 above)
Comment (keith-h) [reply .github/workflows/maven.yml:10]: I thought you'd deleted main

Comment (PhilDigitalJustice) [reply .github/workflows/maven.yml:10]: this is on the saml mock repo, not helm

Comment (keith-h) [reply .github/workflows/maven.yml:10]: Yes.  Got muddled.  Should be main

**Review by PhilDigitalJustice — APPROVED** [review]
LGTM

**Pull request merged**